x-pack/plugins/cloud_security_posture/public/pages/findings/findings_flyout/findings_flyout.tsx

```diff
@@ -107,7 +107,7 @@ export const FindingsRuleFlyout = ({ onClose, findings }: FindingFlyoutProps) =>
           ))}
         </EuiTabs>
       </EuiFlyoutHeader>
-      <EuiFlyoutBody>
+      <EuiFlyoutBody key={tab.id}>
```

Comment:
changing key will force a re-mount of the component
which is the same as wrapping each tab with its own EuiFlyoutBody